war/src/main/webapp/css/style.css

```diff
@@ -1975,3 +1975,23 @@ body.main-panel-only #main-panel {
         padding: 2em 10%;
     }
 }
+
+IMG.icon-sm {
```

Comment:
I agree with @daniel-beck.

@kohsuke We want Jenkins is a modern software, for that reason, we have to look to -at least- to the present :smile: 

From other's exprerience:

> The simplest way to mitigate any potential issues surrounding case sensitivity is to always use lowercase for everything in your markup and CSS, where possible. If that’s not possible, make sure the case you use is consistent between your CSS and your document markup.

http://reference.sitepoint.com/css/casesensitivity